packages/components/src/components/PreferencePage.tsx

```diff
@@ -1,14 +1,38 @@
-import { Footer, Header, PreferenceList } from "@trycourier/react-preferences";
+import { Footer, Header, PreferencesV4 } from "@trycourier/react-preferences";
 import React from "react";
 
 const PreferencePage: React.FunctionComponent<{ draft?: boolean }> = ({
   draft = false,
 }) => {
   return (
-    <div>
-      <Header draft={draft} />
-      <PreferenceList draft={draft} />
-      <Footer draft={draft} />
+    <div
+      style={{
+        height: "100vh",
```

Comment:
any reason for doing inline-styles here insteads of classes?

Reply:
moved it over to styled components!